tests/suites/test_suite_psa_crypto.data

```diff
@@ -6146,3 +6146,51 @@ persistent_key_load_key_from_storage:"":PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY
 
 PSA derive persistent key: HKDF SHA-256, exportable
 persistent_key_load_key_from_storage:"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":PSA_KEY_TYPE_RAW_DATA:1024:PSA_KEY_USAGE_EXPORT:0:DERIVE_KEY
+
+PSA PAKE: invalid alg
+depends_on:PSA_WANT_ALG_ECDSA:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:PSA_WANT_ALG_SHA_256
```

Comment:
`PSA_WANT_ALG_ECJPAKE` dependency is missing. Also in other cases.

Reply:
Also, I'm not sure why this, and other tests below, would depend on `PSA_WANT_ALG_ECDSA`. Perhaps a typo/completion error as you meant `ECJPAKE`?

Reply:
The test cases already depend on `PSA_WANT_ALG_ECJPAKE` in `tests/suites/test_suite_psa_crypto.function`

Reply:
Actually we don't need to depend on `PSA_WANT_ALG_ECJPAKE` here for each test case if the test function already depends on it, which I think it should.

Reply:
Oh ok. Test function has this dependency.